xml/System.Globalization/StringInfo.xml

```diff
@@ -491,9 +491,9 @@ This example uses the <xref:System.Globalization.StringInfo.GetTextElementEnumer
         <Parameter Name="str" Type="System.ReadOnlySpan&lt;System.Char&gt;" Index="0" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="str">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="str">The input span to analyze.</param>
```

Comment:
>The input span to analyze. [](http://example.com/codeflow?start=26&length=26)

The input char span from which to get the first text element length.

Reply:
I ended up leaving this, because the summary explains what it does. If I repeat the summary on this parameter you just get to read both. Note there are many of these here eg `The input string to analyze`